tasks/config/cleanup-config.yml

```diff
@@ -14,3 +14,4 @@
     state: absent
   loop: "{{ nginx_config_files['results'] | map(attribute='files') | sum(start=[]) | map(attribute='path') | list + nginx_config_cleanup_files | default('') | list }}"
   notify: (Handler - NGINX Config) Run NGINX
+  when: "nginx_config_cleanup_paths is defined and 'results' in nginx_config_files.keys()"
```

Comment:
This block should be able to run even when `nginx_config_cleanup_paths` is not defined as long as `nginx_config_cleanup_files` is defined. The conditional needs to be changed to reflect that, and we probably need to add a `default('')` filter to `nginx_config_files['results']` inside `loop` parameter to account for the variable being potentially empty.

Reply:
Humm, In that case it should't really matter if `nginx_config_cleanup_files` is defined, I corrected like that because I didn't know the `default()` filter would slurp the `dict object has no attribute` error and move on.
If you really need it I can add the conditional to the `nginx_config_cleanup_files` othwerwise the amended commit will work just fine.

Reply:
I think it might be best to include the conditional too? That way the task will skip if you forget to specify either of the nginx_config_cleanup variables instead of "silently" fail.